autonomy/cli/utils/click_utils.py

```diff
@@ -90,6 +91,27 @@ def wrapper(f: Callable) -> Callable:
     return wrapper
 
 
+def chain_selection_flag_(
+    default: ChainType = ChainType.LOCAL,
+    mark_default: bool = True,
+) -> Callable:
+    """Flags for abci spec outputs formats."""
```

Comment:
This contains duplicated code from `chain_selection_flag` method we use for wrapping the `from-token` command. I will refactor the `from-token` command definition to use code from the `autonomy.chain` module after we're done implementing `autonomy mint` command group